tools/install.sh

```diff
@@ -1,3 +1,4 @@
+function main {
```

Comment:
For coding style, That has to add parenthesis to main.
before:
```
function main {
```

after:
```
function main() {
```
Check it out: https://github.com/robbyrussell/oh-my-zsh/wiki/Coding-style-guide#functions